.openapi.yaml

```diff
@@ -0,0 +1,266 @@
+openapi: 3.0.2
+info:
+  title: EployeeAPI
+  version: "1.0.0"
+  
+paths:
+  /api/auth:
+    post:
+      description: "Authorization user"
```

Comment:
You haven't marked non idempotent URIs as protected by JWT token

Reply:
+